README.md

```diff
@@ -1,6 +1,12 @@
 # ua-fixer
 
-several user agent fixes for Firefox Android, like making it a phone even if Firefox thinks tablet.
+several user agent fixes for Firefox Android
+
+## features
+
+* forces ua to be a phone, not a table even if Firefox thinks so
```

Comment:
man I would love to run firefox on my table

Reply:
You joke today, but prepare for the IoT of tomorrow.